README.md

```diff
@@ -1,2 +1,4 @@
 # PCR_test
 -miss or mr 郑桑， please do what every you want
+
+- The test result is negative.
```

Comment:
yeahhhhhh! negative

Reply:
looks good

Reply:
here is my repo~
https://github.com/joshua-shuhan/media_computing_test